src/app/pages/HoldRequest.jsx

```diff
@@ -204,12 +204,16 @@ export class HoldRequest extends React.Component {
      */
   renderDeliveryLocation(deliveryLocations = []) {
     const { closedLocations } = this.props;
+    const { openLocationsMust } = appConfig;
     return deliveryLocations.map((location, i) => {
       const displayName = this.modelDeliveryLocationName(location.prefLabel, location.shortName);
       const value = (location['@id'] && typeof location['@id'] === 'string') ?
         location['@id'].replace('loc:', '') : '';
 
-      if (closedLocations.some(closedLocation => displayName.startsWith(closedLocation))) {
+      if (
+        closedLocations.some(closedLocation => displayName.startsWith(closedLocation)) ||
+        (openLocationsMust && !openLocationsMust.some(openLocation => displayName.includes(openLocation)))
```

Comment:
Is there a reason to check location labels over ids? Do we plan to enable multiple locations that include the same location phrase? Will location names ever contain commas? (which we're using to split the env var)

Reply:
These are all basically for convenience: Since we actually see the label on the Hold Request Page, the clearest/most foolproof/most direct way to filter locations is by label (i.e., if I want to have room 315 open, I just put 315 in the list of open locations, and no need to worry about matching ids to labels). Location names might theoretically include commas (I don't think any of them do), and definitely include spaces, and the full names are pretty long, so the easiest thing is to just pick a unique word (like the room number) for each location. In theory there might not be any unique words but in practice it seems there are.

If it seems like this could cause problems we could use id.

Reply:
We probably won't encounter issues with matching on labels. If we do, we can change it to ids. I worry that some phrases will be ambiguous, but in practice it probably won't matter. We have used commas in location names in the past, but don't appear to have any that are now (potentially) open. So I don't see an immediate issue.